### DISCUSSION TITLE
Emails from eNotices2 (NoticeAuthorEmail)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#701] [Q&A] [answered]
We were reviewing, in the preview environment, the notifications sent by email and we see that we do not receive emails to the notice author email when there is a change of status to stopped or published.

When a submit is made we do receive the email.

In the email address used by EU Login for API Key (eSender email address) we do receive all status changes.

Does the preview environment make the same notifications for these two email addresses (noticeAuthorEmail and eSender email)?

Thanks in advance

**Comment by pcolast** [comment, marked as answer]
Good day,

Thank you for contacting Helpdesk for this non-technical issue, where you received an answer in the meantime.

Have a nice day,
Stefan
